bin/openstack-install

```diff
@@ -121,6 +120,8 @@ def parse_options(argv):
     parser.add_argument('--next-charms', dest='next_charms',
                         action='store_true',
                         help="Use /next charms to test upcoming features.")
+    parser.add_argument('--use-nclxd', dest='use_nclxd', action="store_true",
+                        help="Enable LXD support for Nova Compute.")
```

Comment:
Might be nice to have the help mention that it requires --series=vivid. I see we check that later, but if someone's looking at the help we might as well tell them.